src/main/java/nl/b3p/tailormap/api/controller/AppController.java

```diff
@@ -13,7 +13,7 @@
 import nl.b3p.tailormap.api.model.RedirectResponse;
 import nl.b3p.tailormap.api.repository.ApplicationRepository;
 import nl.b3p.tailormap.api.repository.MetadataRepository;
-import nl.b3p.tailormap.api.security.AuthUtil;
```

Comment:
it seems `nl.b3p.tailormap.api.security.AuthUtil` is no longer used anywhere, in that case it can/should be removed